lib/Stampie/Adapter/Guzzle6.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: darren
+ * Date: 18/03/2016
+ * Time: 08:34
+ */
+
+namespace Stampie\Adapter;
+
+
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Psr7\Request;
+
+class Guzzle6 implements AdapterInterface
+{
+    /** @var ClientInterface $client */
+    protected $client;
+
+    /**
+     * Guzzle6 constructor.
```

Comment:
this description is useless. We already know it is a constructor thanks to the method name